spec/system/cohorts/student_report_spec.rb

```diff
@@ -485,4 +486,107 @@
       "This student's access to the course ended on #{time.strftime("%b %-d, %Y")}."
     )
   end
+
+  context "when user is a school admin" do
+    scenario "can access a student report" do
```

Comment:
This scenario seems too large. I expect the contents of this report page to have been tested already - most of what's in here seems to be a dupe of functionality already tested using a coach profile.

Check whether the page loads correctly by testing one or two elements, and check for custom elements shown only to admins, and then move on.